app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -1,59 +1,38 @@
-# For now, this is used to register dummy editions in the content store as
+# For now, this is used to register dummy items in the content store as
```

Comment:
This comment isn't accurate - some subclasses are sending real content items. I think now is as good a time as any to reword it to say that.